planning/autoware_rtc_interface/src/rtc_interface.cpp

```diff
@@ -151,7 +151,7 @@ std::vector<CooperateResponse> RTCInterface::validateCooperateCommands(
 
     const auto itr = std::find_if(
       registered_status_.statuses.begin(), registered_status_.statuses.end(),
-      [command](auto & s) { return s.uuid == command.uuid; });
+      [command](const auto & s) { return s.uuid == command.uuid; });
```

Comment:
@kobayu858 
Are you 100% sure it is OK to make `s` a const reference even though `s` is modified?